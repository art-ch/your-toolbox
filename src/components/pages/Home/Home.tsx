'use client';

import { Hero } from '@/components/Hero/Hero';
import { useTranslation } from 'react-i18next';

export const Home = () => {
  const { t } = useTranslation();

  return (
    <Hero
      title="Your Toolbox"
      subtitle={t('homePage:subtitle')}
      buttonText={t('homePage:buttonText')}
    />
  );
};
