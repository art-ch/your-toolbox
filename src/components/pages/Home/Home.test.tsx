import { render } from '@testing-library/react';

import { Home } from './Home';

describe('Home component', () => {
  it('should render correctly', () => {
    const { container } = render(<Home />);

    expect(container).toMatchSnapshot();
  });
});
