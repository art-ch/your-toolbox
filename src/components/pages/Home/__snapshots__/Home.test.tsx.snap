// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component should render correctly 1`] = `
<div>
  <div
    class="my-0 mx-auto py-40 text-center max-w-lg"
  >
    <div
      class="my-0 mx-auto py-2"
    >
      <h1
        class="text-4xl sm:text-6xl font-semibold leading-none tracking-tight"
      >
        Your toolbox
      </h1>
      <p
        class="py-2"
      >
        Make your computer do the work for you
      </p>
    </div>
    <a
      href="/navigation"
    >
      <button
        class="inline-flex items-center justify-center gap-2 whitespace-nowrap rounded-md text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 [&_svg]:pointer-events-none [&_svg]:size-4 [&_svg]:shrink-0 bg-primary text-primary-foreground hover:bg-primary/90 h-10 px-4 py-2"
      >
        Check available tools
      </button>
    </a>
  </div>
</div>
`;
